frontend/destino-cordoba/src/app/components/user-dashboard-components/user-dashboard-container/user-update-form/user-update-form.component.ts

```diff
@@ -25,19 +37,11 @@ export class UserUpdateFormComponent implements OnInit, OnDestroy {
   ngOnInit(): void {
     const routeParams = this.route.snapshot.paramMap;
     this.userId = Number(routeParams.get("userId"));
-
-    this.subscription = this.userService.getBy(this.userId).subscribe((data) => {
-      this.currentUser = this.formBuilder.group({
-        first_name: data.first_name,
-        last_name: data.last_name,
-        email: data.email,
-        password: "",
-        phone: data.phone,
-        photo: data.photo,
-        address: data.address
-      }); {this.user = data};
-    });
-    //this.userService.getBy(this.userId).subscribe((data) => {this.user = data})
```

Comment:
que no te aparezca los datos del usuario en el formulario es por esto, antes se los asignabas a los valores y ahora solamente inicializas el formulario con los campos vacios

Reply:
Ya veo, yo lo quité porque los values los sacaba de otro objeto, lo dejare como estaba antes.

Reply:
de una, para traerte los datos de address tambien es igual que como hacias antes, pero como al principio van a venir nulos podes usar el ||
this.currentUser = this.formBuilder.group({
    first_name: data.first_name,
    last_name: data.last_name,
    email: data.email,
    password: "",
    phone: data.phone,
    photo: data.photo,
    address: this.formBuilder.group({
      street: data.address.street || '',
      number: data.address.number || '',
      zip_code : data.address.zip_code || ''
    })
  });

Reply:
Lo dejé casi como antes, sigue rompiéndose por eso no lo mergeare xd

Reply:
estuve viendo que haces una llamada a la api para traerte el user, no hace falta, el user ya esta guardado por eso persistimos el inicio de sesion jajaj

      `this.loginService.getCurrentUser().subscribe( (data) => { } );`

Esto haria exactamente lo mismo

Reply:
Ahi estoy modificando django para que el user ya venga con el objeto address, que no lo trae por eso undefined. funciona por ahora pero hay algunos errores, mas tarde te muestro si podes

Reply:
@FrancoGL ahi subi todo a la rama `admin-d`, a mi ya me funciona, probala

Reply:
Si, también me funciona.

Reply:
> estuve viendo que haces una llamada a la api para traerte el user, no hace falta, el user ya esta guardado por eso persistimos el inicio de sesion jajaj
> 
> ```
>   this.loginService.getCurrentUser().subscribe( (data) => { } );
> ```
> 
> Esto haria exactamente lo mismo

Me olvidé de cambiarlo (? xd

Reply:
@FrancoGL ahi mergee la otra rama a esta

Reply:
Dale, mergeo este pull entonces